contrib/test_jenkins.sh

```diff
@@ -596,6 +596,9 @@ run_io_demo() {
 	if [ "X$have_cuda" == "Xyes" ]
 	then
 		mem_types_list+="cuda cuda-managed "
+		../contrib/configure-devel --prefix=$ucx_inst --with-iodemo-cuda
+		$MAKEP
+		$MAKEP install
```

Comment:
in here: config_args+="--with-iodemo-cuda"
before line 609:
`../contrib/configure-devel --prefix=$ucx_inst ${config_args}`
so we build also w/o cuda, and not build if line 604 is false